core/src/apps/bitcoin/authorize_coinjoin.py

```diff
@@ -61,9 +63,14 @@ async def authorize_coinjoin(ctx: wire.Context, msg: AuthorizeCoinJoin) -> Succe
                 )
             )
         text.normal("Maximum total fees:")
+        amount_unit = (
```

Comment:
let's rely on the default from protobuf

Reply:
Fix in 3a1b98050463c515587349cf6abbb4090d6ee8ab